CONTRIBUTING/index.html

```diff
@@ -1,4168 +1,10 @@
-      
-        
-        <a href="../windows/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Windows Container Support" rel="prev">
-          <div class="md-footer__button md-icon">
-            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12z"/></svg>
-          </div>
-          <div class="md-footer__title">
-            <div class="md-ellipsis">
-              <span class="md-footer__direction">
-                Previous
-              </span>
-              Windows Container Support
-            </div>
-          </div>
-        </a>
-      
-      
-        
-        <a href="../running-nix/" class="md-footer__link md-footer__link--next" aria-label="Next: Try Argo using Nix" rel="next">
-          <div class="md-footer__title">
-            <div class="md-ellipsis">
-              <span class="md-footer__direction">
-                Next
-              </span>
-              Try Argo using Nix
-            </div>
-          </div>
-          <div class="md-footer__button md-icon">
-            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4z"/></svg>
-          </div>
-        </a>
-      
-    </nav>
-  
-  <div class="md-footer-meta md-typeset">
-    <div class="md-footer-meta__inner md-grid">
-      <div class="md-copyright">
-  
-  
-    Made with
-    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
-      Material for MkDocs
-    </a>
-  
-</div>
-      
-    </div>
-  </div>
-</footer>
-      
-    </div>
-    <div class="md-dialog" data-md-component="dialog">
-      <div class="md-dialog__inner md-typeset"></div>
-    </div>
-    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.tabs", "navigation.tabs.sticky", "navigation.top"], "translations": {"clipboard.copy": "Copy to clipboard", "clipboard.copied": "Copied to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.placeholder": "Type to start searching", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.term.missing": "Missing", "select.version.title": "Select version"}, "search": "../assets/javascripts/workers/search.22074ed6.min.js"}</script>
-    
-    
-      <script src="../assets/javascripts/bundle.960e086b.min.js"></script>
-      
-    
+  <body>
+    <p>This page has moved to a <a href="https://argo-workflows.readthedocs.io/en/latest/CONTRIBUTING/">
+        https://argo-workflows.readthedocs.io/en/latest/CONTRIBUTING/</a>.</p>
```

Comment:
```suggestion
    <p>This page has moved to <a href="https://argo-workflows.readthedocs.io/en/latest/CONTRIBUTING/">https://argo-workflows.readthedocs.io/en/latest/CONTRIBUTING/</a>.</p>
    <p>You should be redirected there automatically. Please click the link above if you are not redirected.</p>
```

- grammar/typo: unnecessary "a" in "has moved to a"
- formatting/style: one-line the `<p>` element, otherwise it should be multi-lined (as in `<p>\ntext\n</p>`), not two lined.
- add a sentence noting the automatic redirect